_policy/referees.md

```diff
@@ -1,15 +1,448 @@
 ---
-title: Referees Commission Policies
-layout: default
+title: Referees Commission Policy
 
-approval_authority: 
-responsible_officer: 
-first_approved: null
-last_amended: null
+approval_authority: President
+responsible_officer: Sport Development Director
+first_approved: 2010-04-08
```

Comment:
I've based this date on the version [presently published][c] on the website.

[c]: https://www.internationaltouch.org/media/FIT%20Referee%20Policy%20-%20Final%20(08-04-10).pdf